### ISSUE TITLE
Define high level structure for datamesh-platform documentation

### ISSUE THREAD

**Issue by avinashsingh77** [opendatahub-io-contrib/datamesh-platform#45]
This issue aims and identifying how the documentation for this repository should be so that it is easy to follow along for the contributors.
The readme should be structured, detailed and at the same time easy to follow for installation and usage of the platform.
It could have sections like:

- How the platform works
- How to install the platform
- How to build a data pipeline
- others..

**Comment by caldeirav** [comment]
Indeed, thanks for starting this, and I think we should be looking at best examples of content management in other Github repos and typically this means going back to a template (e.g. https://github.com/othneildrew/Best-README-Template) to structure our content and include:

- A shorter overview of the platform (we can move detailed content to an architecture documentation page)
- Dependencies and Install instructions (starting with OpenShift deployment)
- Quickstart (I can take care of this, it will be based on the data-product-template repo but it makes sense to have the instructions here)
- Roadmap (link to the project page)
- How to contribute
